requirements-test.txt

```diff
@@ -20,3 +20,5 @@ pytest-cov==2.3.1 \
 #    --hash=sha256:fa0a212283cdf52e2eecc24dd6459bb7687cc29adb60cb84258fab73be8dda0f
 pytest-xdist==1.15.0 \
 #    --hash=sha256:6238395f8bd050f9288a3b10f34330edece80f4424cf2b4204d6e7d622f0f00b
+requests==2.12.4
+lxml==3.7.2
```

Comment:
Thanks for pinning these, can you specify hashes as well, like we do for other packages?